soap/serializers.py

```
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+"""
+سریالایزرهای اپ SOAP
+"""
+
+from typing import Any, Dict
```

Ruff (0.12.2)

Module level import not at top of file

(E402)